Makefile

```diff
@@ -3,9 +3,10 @@ LLVM_SUFFIX?=-12
 CLANG?=clang$(LLVM_SUFFIX)
 CFLAGS=-O2
 JAVA?=$(shell test/findJava.sh)
-TARGETS=all clean test
+TARGETS=all clean test pack extra
 export CLANG CFLAGS BAL JAVA LLVM_SUFFIX
-SUBDIRS=compiler.d runtime.d test.d
+SUBDIRS=compiler.d runtime.d test.d extra.d
+EXTRAS=regex
```

Comment:
Don't see why you need EXTRAS or extras here? Can't you just use extra.d?

Reply:
What I want to do is run `make regex` (or any other extra in the future)  from the root and be able to build just that (while retaining the ability to build all the extras in one command).